<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VtkDemoClass</class>
 <widget class="QMainWindow" name="VtkDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VtkDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QVTKWidget" name="qvtkWidget3"/>
     </item>
     <item row="0" column="0">
      <widget class="QVTKWidget" name="qvtkWidget1"/>
     </item>
     <item row="0" column="1">
      <widget class="QVTKWidget" name="qvtkWidget2"/>
     </item>
     <item row="1" column="1">
      <widget class="QVTKWidget" name="qvtkWidget4"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="VtkSetValueBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>221</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>操作视窗</string>
    </property>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>281</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ThreeViewPropSetTab">
      <attribute name="title">
       <string>三维视图</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="MarchingCubeSetPropsTabBox">
      <attribute name="title">
       <string>面绘制</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="RayCastingSetPropsTab">
      <attribute name="title">
       <string>体绘制</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="VtkPropsBox">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>属性视窗</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>191</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="VtkPrpsTab">
      <attribute name="title">
       <string>属性值</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="VtkDemo.qrc"/>
 </resources>
 <connections/>
</ui>
